src/app/collection/pages/nice-challenge/components/footer/footer.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="wrapper">
+  <div class="rights-container">
+    <div class="rights-text-box">
+      <p>
+        This is the page for the NICE Challenge Project curriculum 
+        collection on CLARK 
+        (<a href="www.clark.center" aria-label="Link to CLARK home page">
+        www.clark.center</a>). While CLARK hosts the curiculum all 
+        rights are retained by NICE. Learn more at 
+        <a 
+          href="https://nice-challenge.com/"
+          aria-label="Link to NICE challenge home page"
+        >
+          https://nice-challenge.com/
+        </a>  
+      </p>
+    </div>
+  </div>
+  <div class="logo-container">
+    <img 
+      src="../../../../../../assets/images/collections/NICE_logo-wo_symbol.png" 
+      alt="NICE Logo"
```

Comment:
Are their any other options for the Nice logos? It would be nice not to have the white background.